src/gdcmodels/init_index.py

```diff
@@ -38,6 +39,12 @@ def get_parser() -> ArgumentParser:
         nargs="+",
         help="index to be initialized",
     )
+    parser.add_argument(
```

Comment:
Food for thought, an index has an alias attribute. Aliases are not a first order structure in ES. If an index is removed that alias is also removed. Does this account for indices that share the same alias?

Reply:
I'm not sure if I'm understanding correctly but if the indices are removed the aliases also go with them.

Reply:
If the last index using an alias is removed, then the alias no longer available.